cmd/root.go

```diff
@@ -41,6 +46,8 @@ func Execute() {
 
 func init() {
 	rootCmd.SetHelpCommand(&cobra.Command{Use: "no-help", Hidden: true})
+	rootCmd.PersistentFlags().StringVarP(&FileFlag, "file", "f", config.OneBuildConfigFileName,
```

Comment:
Don't assign value to `FileFlag` instead just bind the flags with viper and you can access them any time.

Reply:
Ok good idea.